pkg/api/utils/apiUtils.go

```diff
@@ -53,28 +53,28 @@ func (p *APIHandler) getHTTPClient() *http.Client {
 }
 
 // SendEvent sends an event to Keptn
-func (e *APIHandler) SendEvent(event models.KeptnContextExtendedCE) (*models.EventContext, *models.Error) {
+func (p *APIHandler) SendEvent(event models.KeptnContextExtendedCE) (*models.EventContext, *models.Error) {
```

Comment:
why "p" ?

Reply:
This was suggested by GoLand, as all other receivers in this file are called `p

Reply:
I think there are several conventions, in this case:
`a` or `ah` I also see `A` sometimes.
Maybe in the future, we'll have a list of coding guidelines/conventions for keptn development ?

Reply:
I've changed it to `a`